acnportal/acnsim/simulator.py

```diff
@@ -264,7 +268,7 @@ def to_dict(self, context_dict=None):
         attribute_dict['start'] = self.start.strftime('%H:%M:%S.%f %d%m%Y')
 
         try:
-            base.json.dumps(self.signals)
+            json.dumps(self.signals)
```

Comment:
Since signals is a dictionary, does it make sense to try to serialize each entry rather than all at once? It may be that some entries are serializable while some are not...

Reply:
Maybe leave this for a future PR as a feature of sorts.

Reply:
I agree with leaving this as part of #56. Then, we will need to serialize each value individually as it'll be a data structure similar to ev_history.